srcpkgs/mono-addins/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'mono-addins'
+pkgname=mono-addins
+version=1.3
+revision=1
+wrksrc=${pkgname}-${pkgname}-${version}
+build_style=gnu-configure
+hostmakedepends="autoconf automake pkg-config"
+makedepends="mono-devel gtk-sharp2"
```

Comment:
also needs `depends="mono gtk-sharp2"`

Reply:
`mono-addins` builds without these in `depends`

Reply:
> `mono-addins` builds without these in `depends`

Those are runtime dependencies, not needed during build.

Reply:
Ah, I see.